src/ddo/interfaces/Status.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Status attributes of Assets Metadata.
+ * @see https://github.com/oceanprotocol/OEPs/tree/master/8
+ */
+export interface Status {
+  /**
+   * Use to flag unsuitable content. True by default. If it's false, the content must not be returned.
+   * @type {boolean}
+   * @example true
+   */
+  isListed?: boolean
```

Comment:
from the comment this looks like `isInPurgatory`, or what is the purpose? @kremalicious @alexcos20

Reply:
we decided for now that we are going to use only isOrderDisabled